Core/include/Acts/Geometry/GenericCuboidVolumeBounds.hpp

```diff
@@ -89,6 +89,14 @@ class GenericCuboidVolumeBounds : public VolumeBounds {
                                   const Vector3& envelope = {0, 0, 0},
                                   const Volume* entity = nullptr) const final;
 
+  /// Get the canonical binning values, i.e. the binning values
+  /// for that fully describe the shape's extent
+  ///
+  /// @return vector of canonical binning values
+  std::vector<Acts::BinningValue> canonicalBinning() const override {
```

Comment:
Do you want the `(x,y,z)` into the base class, so it is clear that classes that do now overload this get just the basic checks?